### PR TITLE
adding minutes from April 4th mentoring meeting

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: subcommittees/mentoring/minutes-2016-04-04.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+* Kate Hertweck
+* Raniere Silva
+ 
+ 
+## Agenda
+* Confirm agenda + choose note-taker
+* Data Gathering
+	* if you haven't already, please fill out this form: https://docs.google.com/forms/d/1eDLPShSyQPJKZ8jRr8VHQSaV_CxVvQOdGTSJHo6dOWg/viewform
+	* and this spreadsheet: https://docs.google.com/spreadsheets/d/1in5KnkD8H0XQ-IwZM4fe2rYNPEmvr2miT13ElKfmkD4/
+* General discussion of vision and goals for the committee, incl,
```
Comment (rgaiacs) [inline subcommittees/mentoring/minutes-2016-04-04.md]: "incl" -> "include"?

File: subcommittees/mentoring/minutes-2016-04-04.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	* relationship w/ instructor training, including merging of various help/discussion sessions
```
Comment (rgaiacs) [inline subcommittees/mentoring/minutes-2016-04-04.md]: "w/" -> "with"

**Comment by rgaiacs** [comment]
@ChristinaLK or @raynamharris Could you try clear the minutes to make easy for those that didn't attend the meeting read it? Thanks.

**Comment by ChristinaLK** [comment]
Is this better?

**Comment by raynamharris** [comment]
@ChristinaLK yes, this looks great.

**Pull request merged**